### PR TITLE
Tcpip integration

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

File: Integrations/integration-TCPIPUtils.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+description: enrich information from tcipip.com
```
Comment (dorsha) [inline Integrations/integration-TCPIPUtils.yml]: Should start with capital letter

File: Integrations/integration-TCPIPUtils.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+        sys.exit(0)
+
+    # You can use demisto.args()[argName] to get a specific arg. args are strings.
+    # You can use demisto.params()[paramName] to get a specific params.
+    # Params are of the type given in the integration page creation.
+  type: python
+  commands:
+  - name: ip
+    arguments:
```
Comment (dorsha) [inline Integrations/integration-TCPIPUtils.yml]: Missing description for argument, take a look at other ips integrations (like ipinfo)

File: Integrations/integration-TCPIPUtils.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        "HumanReadable": tableToMarkdown("TCPIPUtils Info for " + ip_address, [hm], "")})
+
+        sys.exit(0)
+
+    # You can use demisto.args()[argName] to get a specific arg. args are strings.
+    # You can use demisto.params()[paramName] to get a specific params.
+    # Params are of the type given in the integration page creation.
+  type: python
+  commands:
+  - name: ip
```
Comment (dorsha) [inline Integrations/integration-TCPIPUtils.yml]: Missing description for command, take a look at other ips integrations (like ipinfo)

File: Integrations/integration-TCPIPUtils.yml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    outputs:
+    - contextPath: IP.Address
+      description: ip address
+    - contextPath: IP.ASN
+      description: ip asn
+    - contextPath: IP.Geo.Country
+      description: ip country
+    - contextPath: IP.Geo.Description
+      description: ip city
+    description: get information for ip
```
Comment (dorsha) [inline Integrations/integration-TCPIPUtils.yml]: Description should be better (take a look at other integrations) and should start with capital letter

File: Integrations/integration-TCPIPUtils.yml [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    # You can use demisto.params()[paramName] to get a specific params.
+    # Params are of the type given in the integration page creation.
+  type: python
+  commands:
+  - name: ip
+    arguments:
+    - name: ip
+      required: true
+    outputs:
+    - contextPath: IP.Address
```
Comment (dorsha) [inline Integrations/integration-TCPIPUtils.yml]: descriptions of outputs should be like in ipinfo

File: Integrations/integration-TCPIPUtils.yml (hunk 1 above)
Comment (dorsha) [reply Integrations/integration-TCPIPUtils.yml]: And the description should be better (something like ipinfo)

**Comment by erezh31** [comment]
@dorsha I fixed the comments

**Pull request merged**